lib/open_food_network/reports.rb

```diff
@@ -0,0 +1,5 @@
+module OpenFoodNetwork
```

Comment:
In the discussion about engines, we talked about classes for conventions among domain-engines which we could put in a gem or engine. I prefer engine and not gem, because of the bigger overhead in maintaining a gem. Perhaps we should already move this to that new engine? We can do that in a separate PR.

When we move forward and want to separate each domain-engine into a different application, we can extract this engine into a gem, and each domain-engine application can upgrade the gem at its own pace.

Reply:
first, guys, it feels so good we are having this conversation!!!

I agree with the reasoning about creating a common engine and not a gem as long as this engine does not have models in it, just services. we can call this engine "commons" :-)

But as general YAGNI, I don't think we should abstract things into that engine until we have 2 engines needing it. We may end up having all reports in this order_management engine (most of them belong here right?) or even end up moving all reports to a separate engine of it's own...
so, I am ok with (1) creating that commons engine now with common reports stuff in it, but for now (2) I'd just move these things into the orders_management engine /app/services/reports folder. your choice (what is important is to stop using open_food_network namespace).

Reply:
Agree with @luisramos0 on 2

Reply:
Sounds great, @luisramos0. :-) I agree too.